pcsc-sys/build.rs

```diff
@@ -20,15 +20,24 @@ fn main() {
         }
 
         _ => {
-            pkg_config::Config::new()
-                .atleast_version("1")
-                .probe("libpcsclite")
-                .expect(&format!(
-                    r#"Could not find a PCSC library.
+            let lib_dir = env::var("PCSCLITE_LIB_DIR");
+            let include_dir = env::var("PCSCLITE_INCLUDE_DIR");
+
+            if lib_dir.is_ok() && include_dir.is_ok() {
```

Comment:
Would be better to use a `match` here, instead of checking `is_ok()` and then `unwrap()`.

Reply:
Agreed, but since we can drop the include directory, an `if let` would do as well.
Do you have any preference ?